services/ui-src/src/components/pages/HelpPage/HelpPage.test.tsx

```diff
@@ -16,14 +17,48 @@ describe("Test HelpPage", () => {
   });
 
   test("Check that HelpPage renders", () => {
-    expect(screen.getByText(verbiage.intro.header)).toBeVisible();
+    expect(screen.getByText(verbiage.intro.header)).toHaveTextContent(
+      "How can we help you?"
+    );
+  });
+
+  test("Check for email links", () => {
+    const email1 = screen.getByRole("link", { name: "mdct_help@cms.hhs.gov" });
+    expect(email1).toHaveAttribute("href", "mailto:mdct_help@cms.hhs.gov");
+    const email2 = screen.getByRole("link", { name: "MFPDemo@cms.hhs.gov" });
+    expect(email2).toHaveAttribute("href", "mailto:MFPDemo@cms.hhs.gov");
+  });
+
+  test("Question 1: When Expand button clicked it collapses", async () => {
```

Comment:
I think this is a good test with the wrong name - if I'm reading it right, we're testing that it expands. Same with the test for Question 2.

There are also some `act()` calls in here that can probably be removed.

```suggestion
  test("Question 1: When Expand button clicked it expands", async () => {
```